### PR TITLE
Add minutes for meeting on 2017-10-13

### PR DESCRIPTION
**Pull request by NicholasRostant**
Please give a quick proofread before merging

### CODE REVIEW

File: general/2017-10-13/minutes.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+- Investigate Christmas meal
+
+### Jack (jaw84) 
+- Preliminary organisation of FOSDEM trip
+- Guage interest in trip
+- Follow up on hoodies when finances sorted
+
+### Claudio (clc20) 
+- Preliminary organisation of FOSDEM trip
+- Guage interest in trip
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: Guage -> Gauge

File: general/2017-10-13/minutes.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+### Nick (ndr2) 
+- Preliminary organisation of FOSDEM trip
+- Guage interest in trip
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: Guage -> Gauge

File: general/2017-10-13/minutes.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+- Guage intrest in Hacktoberfest event
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: Guage -> Gauge

File: general/2017-10-13/minutes.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+## List of all actions:
+
+1. Inform Union of election result (sap21)
+2. Follow up on Beechings sponsorship (klo2, alt38)
+3. Invesigate more potential sponsors (klo2, alt38)
+4. Preliminary organisation of FOSDEM trip (ndr2, jaw84, clc20)
+5. Guage interest in trip (ndr2, jaw84, clc20)
+6. Investigate Christmas meal (klo2, alt38)
+7. Follow up on hoodies when finances sorted (jaw84) 
+8. Guage interest in Hactoberfest event (ndr2)
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: Guage -> Gauge

File: general/2017-10-13/minutes.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+    
+
+# Actions:
+## List of all actions:
+
+1. Inform Union of election result (sap21)
+2. Follow up on Beechings sponsorship (klo2, alt38)
+3. Invesigate more potential sponsors (klo2, alt38)
+4. Preliminary organisation of FOSDEM trip (ndr2, jaw84, clc20)
+5. Guage interest in trip (ndr2, jaw84, clc20)
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: Guage -> Gauge

File: general/2017-10-13/minutes.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+## Item 4 - FOSDEM
+- Nick, Jack and Claudio to organise.
+- Positive experience at Condo Gardens last year could be an option for accomodation.
+- The union needs to be informed early
+- Target to finalise plans by December
+- Noted that the minibusses cannot be taken out of the country
+- Possible transport options: flights, trains, coaches or personal vehicles.
+- Mentioned that some members may want to travel sepearately
+- Google Form to be sent around to guage interest from members and assist in planning.
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: guage -> gauge

File: general/2017-10-13/minutes.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+- Noted that the union can sponsor societies if they make use of facilities.
+
+## Item 4 - FOSDEM
+- Nick, Jack and Claudio to organise.
+- Positive experience at Condo Gardens last year could be an option for accomodation.
+- The union needs to be informed early
+- Target to finalise plans by December
+- Noted that the minibusses cannot be taken out of the country
+- Possible transport options: flights, trains, coaches or personal vehicles.
+- Mentioned that some members may want to travel sepearately
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: sepearately -> separately

File: general/2017-10-13/minutes.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+
+- Nothing to report
+
+### Carried Forward:
+- Nothing to report
+
+## Item 1 - Election of Welfare Officer 
+Union has required a mandatory position of Welfare Officer to look after member wellbeing.
+- Stephen Johnstone
+    - Only canidate nominated was Stephen Johnstone, seconded by Stephen Coiner
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: canidate  -> candidate

File: general/2017-10-13/minutes.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+- Inform Union of election result 
+
+### Klaudia (klo2)
+- Follow up on Beechings sponsorship
+- Invesigate more potential sponsors
+- Investigate Christmas meal
+
+### Alex (alt38)
+- Follow up on Beechings sponsorship
+- Invesigate more potential sponsors
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: Invesigate -> Investigate

File: general/2017-10-13/minutes.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+## Actions split by member:
+
+### All:
+
+### Sam (sap21):
+- Inform Union of election result 
+
+### Klaudia (klo2)
+- Follow up on Beechings sponsorship
+- Invesigate more potential sponsors
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: Invesigate -> Investigate

File: general/2017-10-13/minutes.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+  - Nick has indicated interest in running a hacktoberfest event
+  - Introduce people to open source and earn a free t-shirt
+    
+
+# Actions:
+## List of all actions:
+
+1. Inform Union of election result (sap21)
+2. Follow up on Beechings sponsorship (klo2, alt38)
+3. Invesigate more potential sponsors (klo2, alt38)
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: Invesigate -> Investigate

File: general/2017-10-13/minutes.md [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+## Item 4 - FOSDEM
+- Nick, Jack and Claudio to organise.
+- Positive experience at Condo Gardens last year could be an option for accomodation.
+- The union needs to be informed early
+- Target to finalise plans by December
+- Noted that the minibusses cannot be taken out of the country
+- Possible transport options: flights, trains, coaches or personal vehicles.
+- Mentioned that some members may want to travel sepearately
+- Google Form to be sent around to guage interest from members and assist in planning.
+- Disscussion to be held in #fosdem channel on Slack.
```
Comment (jackwebb6) [inline general/2017-10-13/minutes.md]: Disscussion -> Discussion

**Comment by NicholasRostant** [comment]
Cheers I'll sort those tonight

File: general/2017-10-13/minutes.md [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+
+# Actions:
+## List of all actions:
+
+1. Inform Union of election result (sap21)
+2. Follow up on Beechings sponsorship (klo2, alt38)
+3. Invesigate more potential sponsors (klo2, alt38)
+4. Preliminary organisation of FOSDEM trip (ndr2, jaw84, clc20)
+5. Guage interest in trip (ndr2, jaw84, clc20)
+6. Investigate Christmas meal (klo2, alt38)
```
Comment (sap218) [inline general/2017-10-13/minutes.md]: Was Christmas meal my job?

**Comment by NicholasRostant** [comment]
Should be good now

**Pull request merged**